NavigationSystem/Hardwares/CAN_Services/CANService.cpp

```diff
@@ -54,12 +55,12 @@ CanMsg CANService::getCANMessage()
   return Cmsg;
 }
 
-std::future<void> CANService::start() 
+std::future<void> CANService::start()
 {
-  
+
   m_Running.store(true);
   wiringPiSetup();
-  int SPISpeed = 1000000;
+  int SPISpeed = 5000000;
```

Comment:
Asked Viktor L about this, he said it is the clock frequency of the wiring PI, should be now problem to lower so I changed it back to 1000000